pulsar-io/jdbc/core/src/main/java/org/apache/pulsar/io/jdbc/JdbcAbstractSink.java

```diff
@@ -334,7 +333,12 @@ private void internalFlush(Deque<Record<T>> swapList) throws SQLException {
         }
     }
 
-    private void internalFlushBatch(Deque<Record<T>> swapList, PreparedStatement currentBatch, int count, long start) throws SQLException {
+    private void internalFlushBatch(
+            Deque<Record<T>> swapList,
+            PreparedStatement currentBatch,
+            int count,
+            long start
+    ) throws SQLException {
```

Comment:
It seems to be repeated with  #18146.

Reply:
@coderzc I think we should proceed with this patch since it adds a CI task to prevent regression.

Reply:
But it's OK I do some rebases.